packages/tasks/src/tasks/any-to-any/about.md

```diff
@@ -0,0 +1,58 @@
+## Use Cases
+
+### Embodied Agents
+
+Any-to-any models can help embodied agents operate in multi-sensory environments, such as video games or physical robots. The model can take in an image or video of a scene, text prompts, and audio, and respond by generating text, actions, predict next frames, or generate speech commands.
+
+### Real-time Accessibility Systems
+
+Any-to-any model can be used aid visually impaired people. A real-time on-device any-to-any model can take a real-world video stream from wearable glasses, and describe the scene in audio (e.g., "A person in a red coat is walking toward you") or provide real-time closed captions and environmental sound cues.
```

Comment:
nit: it's good to not start each line of the page with `Any-to-any`  - imo we can rephrase so that they sound different.

Reply:
brb replacing with AGI 
![image](https://github.com/user-attachments/assets/ddf0b40f-17c0-49f9-932d-26c2b1cfca5f)